setup.py

```diff
@@ -45,6 +45,7 @@
         ]
     },
     install_requires=[
+        "fmu-ensemble",
```

Comment:
How is the minimum version handled when not specified?

Reply:
Then there is no minimum required version. Do we currently require what is today's latest version? I can add a version specifier.

Reply:
Will at least require the version where get_meta_data was added soon, which is pretty much the current version